backend/plonk/bls24-315/prove.go

```diff
@@ -55,7 +55,7 @@ type Proof struct {
 	H [3]kzg.Digest
 
 	// PI2, the BSB22 commitment
-	PI2 kzg.Digest
+	PI2 []kzg.Digest
```

Comment:
Should we find a better name for those commitments? What does the '2' means, is it to tell that a new public input is derived per commitment ?

Reply:
Yes I'm in favor of a better name! PI2 came from how in the original paper the vector/polynomial holding the public values was called PI.